projects/demo/src/modules/components/avatar/avatar.component.ts

```diff
@@ -37,7 +37,7 @@ export class ExampleTuiAvatarComponent {
 
     avatarUrl = null;
 
-    text = 'daenerys targaryen';
+    text = 'Daenerys Targaryen';
```

Comment:
This was intentional to see that it capitalizes first letters

Reply:
Oh, thought about that. Got it!